lxd/instance/instancetype/instance.go

```diff
@@ -372,6 +372,19 @@ var InstanceConfigKeysAny = map[string]func(value string) error{
 		return err
 	},
 
+	// lxdmeta:generate(entities=instance; group=miscellaneous; key=ubuntu_pro.guest_attach)
+	// Indicate whether the guest should auto-attach Ubuntu Pro at start up.
+	// The allowed values are `off`, `on`, and `available`.
+	// If set to `off`, it will not be possible for the Ubuntu Pro client to obtain guest token via devlxd.
+	// If set to `available`, attachment via guest token is possible but will not be performed automatically by the Ubuntu Pro client at startup.
+	// If set to `on`, attachment will be performed automatically by the Ubuntu Pro client at startup.
```

Comment:
Probably should add here a mention that the Ubuntu Pro client is required to be installed inside the guest.